packages/fast-colors/src/named-colors.ts

```diff
@@ -0,0 +1,896 @@
+import { ColorRGBA64Config } from "./color-rgba-64";
+
+// https://www.w3schools.com/colors/colors_names.asp
```

Comment:
nit: can we link to https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#Color_keywords instead?